rastervision/evaluation/semantic_segmentation_evaluation.py

```diff
@@ -12,6 +12,40 @@ def __init__(self, class_map):
         super().__init__()
         self.class_map = class_map
 
+    def compute_vector(self, gt_filename, pred_filename, mode, class_id):
+        import mask_to_polygons.vectorification as vectorification
+        import mask_to_polygons.processing.score as score
+
+        ground_truth = vectorification.geometries_from_geojson(gt_filename)
+        predictions = vectorification.geometries_from_geojson(pred_filename)
+        if len(ground_truth) > 0 and len(predictions) > 0:
+            results = score.spacenet(predictions, ground_truth)
+
+            true_positives = results['tp']
+            false_positives = results['fp']
+            false_negatives = results['fn']
+            precision = float(true_positives) / (
+                true_positives + false_positives)
+            recall = float(true_positives) / (true_positives + false_negatives)
+            if precision + recall != 0:
+                f1 = 2 * (precision * recall) / (precision + recall)
+            else:
+                f1 = 0.0
+            count_error = int(false_positives + false_negatives)
+            gt_count = len(ground_truth)
+            class_name = 'vector-{}'.format(mode)
```

Comment:
You should also include the `class_name` in addition to the `mode`.

Reply:
Done.